tidb.go

```diff
@@ -212,7 +212,7 @@ func GetHistory(ctx sessionctx.Context) *StmtHistory {
 }
 
 // GetRows4Test gets all the rows from a RecordSet, only used for test.
-func GetRows4Test(goCtx goctx.Context, ctx sessionctx.Context, rs ast.RecordSet) ([]types.Row, error) {
+func GetRows4Test(ctx sessionctx.Context, goCtx goctx.Context, rs ast.RecordSet) ([]types.Row, error) {
```

Comment:
go context should be the first parameter.

Reply:
now we can just `import "golang.org/x/net/context"` instead of `import goctx "golang.org/x/net/context"`

Reply:
I see, context.Context should be the first parameter of a function (golint).

All goCtx are aliases for "golang.org/x/net/context". Can I replace "golang.org/x/net/context" with "context"?

Reply:
`import "context"` instead of `import goctx "golang.org/x/net/context"`

Reply:
I'm not sure, @tiancaiamao PTAL